pkg/agent/controller/noderoute/node_route_controller_test.go

```diff
@@ -445,3 +475,160 @@ func TestCreateIPSecTunnelPortCert(t *testing.T) {
 		})
 	}
 }
+
+func TestGetNodeMAC(t *testing.T) {
+	validMac, err := net.ParseMAC("00:1B:44:11:3A:B7")
+	require.NoError(t, err)
```

Comment:
```suggestion
	validMac, _ := net.ParseMAC("00:1B:44:11:3A:B7")
```
no need to check error for known input which can never fail.